cmd/svcat/versions/version_cmd.go

```diff
@@ -71,6 +71,7 @@ func (c *versionCmd) version() error {
 		client = pkg.VERSION
 	}
 
+	output.WriteVersion(c.Output, client, "")
```

Comment:
Do you think i should split `output.WriteVersion` into `output.WriteClientVersion` and `output.WriteServervVersion` ?

Reply:
good idea! Yes that would work well.